### PR TITLE
Minor edits to README, LICENSE, setup.cfg, setup.py; Entry about 2.20…

### PR DESCRIPTION
**Pull request by rnikutta**
….0 tagged release in RELEASE_NOTES.md

### CODE REVIEW

**Comment by rnikutta** [comment]
@mjfitzpatrick If you could please review this minor final PR, and merge with master, then I'll tag a 2.20.0 release, and build the pypi package. Thanks!

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# Data Lab 2.20.0 release notes [20211201]
+
+Tagging this as a release, with most changes made such that clients
+call services at the noirlab.edu domain. The old noao.edu domain
+expired on Dec 29, 2021.
```
Comment (mjfitzpatrick) [inline RELEASE_NOTES.md:5]: Nov 29, 2021

**Review by mjfitzpatrick — CHANGES_REQUESTED** [review]
RELEASE_NOTES.md has the wrong noao.edu expiry date, should be Nov 29, 2021

File: RELEASE_NOTES.md (hunk 1 above)
Comment (mjfitzpatrick) [reply RELEASE_NOTES.md]: I changed the date

**Pull request merged**